WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerAutomatticTracks.m

```diff
@@ -557,6 +557,26 @@ + (TracksEventPair *)eventPairForStat:(WPAnalyticsStat)stat
         case WPAnalyticsStatGravatarUploaded:
             eventName = @"me_gravatar_uploaded";
             break;
+            
+        case WPAnalyticsStatGutenbergWarningConfirmDialogShown:
+            eventName = @"gutenberg_warning_confirm_dialog_shown";
+            break;
+        case WPAnalyticsStatGutenbergWarningConfirmDialogShownYesTapped:
+            eventName = @"gutenberg_warning_confirm_dialog_shown_yes_tapped";
+            break;
+        case WPAnalyticsStatGutenbergWarningConfirmDialogShownCancelTapped:
+            eventName = @"gutenberg_warning_confirm_dialog_shown_cancel_tapped";
+            break;
+        case WPAnalyticsStatGutenbergWarningConfirmDialogShownDontShowAgainChecked:
+            eventName = @"gutenberg_warning_confirm_dialog_shown_dont_show_again_checked";
+            break;
+        case WPAnalyticsStatGutenbergWarningConfirmDialogShownDontShowAgainUnchecked:
+            eventName = @"gutenberg_warning_confirm_dialog_shown_dont_show_again_unchecked";
+            break;
+        case WPAnalyticsStatGutenbergWarningConfirmDialogShownLearnMoreTapped:
+            eventName = @"gutenberg_warning_confirm_dialog_shown_learn_more_tapped";
```

Comment:
Since the tracks naming convention is `context_source_action`, maybe remove the `_shown_` component from all but `gutenberg_warning_confirm_dialog_shown` where its the action being performed.

Reply:
cc @mzorz

Reply:
PR ready here @hypest https://github.com/wordpress-mobile/WordPress-Android/pull/8541 - let's wait for that PR to be merged first, then I'll update the other branch  before we continue with https://github.com/wordpress-mobile/WordPress-Android/pull/8539 👍